interface/super/edit_globals.php

```diff
@@ -645,6 +645,38 @@ function checkBackgroundServices()
                                                     echo "</option>\n";
                                                 }
                                                 echo "  </select>\n";
+                                            } elseif ($fldtype == GlobalSetting::DATA_TYPE_MULTI_DASHBOARD_CARDS) {
+                                                $hiddenList = [];
+                                                $ret = sqlStatement("SELECT gl_value FROM `globals` WHERE `gl_name` = 'hide_dashboard_cards'");
+                                                while ($row = sqlFetchArray($ret)) {
+                                                    $hiddenList[] = $row['gl_value'];
+                                                }
+                                                // The list of cards to hide. For now add to array new cards.
+                                                $res = array(
+                                                    ['card_abrev' => '', 'card_name' => xlt('None or Reset')],
+                                                    ['card_abrev' => attr('card_allergies'), 'card_name' => xlt('Allergies')],
+                                                    ['card_abrev' => attr('card_amendments'), 'card_name' => xlt('Amendments')],
+                                                    ['card_abrev' => attr('card_disclosure'), 'card_name' => xlt('Disclosures')],
+                                                    ['card_abrev' => attr('card_lab'), 'card_name' => xlt('Labs')],
+                                                    ['card_abrev' => attr('card_medicalproblems'), 'card_name' => xlt('Medical Problems')],
+                                                    ['card_abrev' => attr('card_medication'), 'card_name' => xlt('Medications')],
+                                                    //['card_abrev' => 'card_rx', 'card_name' => 'Prescriptions'], // For now don't hide because can be disabled as feature.
```

Comment:
up to you, but i think ok if you want to also include prescriptions in this feature

Reply:
@bradymiller prescriptions already have its own disable feature. No need to include it here.

Reply:
But I think I see @bradymiller point is to move the old turn-off feature into here so that people don't have to go to two spots to turn off a card.

Reply:
I agree but I left to others to decide and I asked admins on signal is why Brady chimed in. but i'll add in my PR. thanks